examples/tour.jl

```diff
@@ -52,35 +58,45 @@ spo2_signal = Signal(channel_names=[:spo2],
 # an aside: The hypothetical person from which these hypothetical signals were
 # hypothetically recorded must be experiencing some pretty crazy pathologies if
 # their EEG/ECG are just saw waves...
-saws(signal, seconds) = [(j + i) % 100 * signal.sample_resolution_in_unit for
-                         i in 1:channel_count(signal),
-                         j in 1:(signal.sample_rate * seconds)]
+saws(signal) = [(j + i) % 100 * signal.sample_resolution_in_unit for
+                i in 1:channel_count(signal), j in 1:sample_count(signal)]
 
 # The second argument in the `Samples` constructor is a `Bool` that specifies if
 # the data is in its encoded representation. Here, we construct our signals as
 # "decoded" (i.e. in actual units, though for this toy example it doesn't really
 # matter) and then "encode" them according to the specified:
-eeg = encode(Samples(eeg_signal, false, saws(eeg_signal, 20)))
-ecg = encode(Samples(ecg_signal, false, saws(ecg_signal, 20)))
-spo2 = encode(Samples(spo2_signal, false, saws(spo2_signal, 20)))
+eeg = encode(Samples(eeg_signal, false, saws(eeg_signal)))
+ecg = encode(Samples(ecg_signal, false, saws(ecg_signal)))
+spo2 = encode(Samples(spo2_signal, false, saws(spo2_signal)))
 
 # Here are some basic functions for examining `Samples` instances:
-@test sample_count(eeg) == eeg.signal.sample_rate * 20
+@test sample_count(eeg) == sample_count(eeg_signal) == 20 * eeg_signal.sample_rate
 @test channel_count(eeg) == channel_count(eeg_signal) == 19
 @test channel(eeg, :f3) == channel(eeg_signal, :f3) == 2
 @test channel(eeg, 2) == channel(eeg_signal, 2) == :f3
-@test duration(eeg) == duration(ecg) == duration(spo2) == Second(20)
+@test duration(eeg) == duration(span(eeg_signal)) == Second(20)
 
 # Here are some basic indexing examples using `getindex` and `view` wherein
 # channel names and sample-rate-agnostic `TimeSpan`s are employed as indices:
-span = TimeSpan(Second(3), Second(9))
-span_range = index_from_time(eeg.signal.sample_rate, span)
-@test eeg[:, span].data == view(eeg, :, span_range).data
+slice_span = TimeSpan(Second(3), Second(9))
+span_range = index_from_time(eeg.signal.sample_rate, slice_span)
+@test eeg[:, slice_span].data == view(eeg, :, span_range).data
 @test eeg[:f3, :].data == view(eeg, 2, :).data
 @test eeg[:f3, 1:10].data == view(eeg, 2, 1:10).data
-@test eeg[:f3, span].data == view(eeg, 2, span_range).data
+@test eeg[:f3, slice_span].data == view(eeg, 2, span_range).data
 @test eeg[[:f3, :c3, :p3], 1:10].data == view(eeg, 2:4, 1:10).data
-@test eeg[[:c3, 4, :f3], span].data == view(eeg, [3, 4, 2], span_range).data
+@test eeg[[:c3, 4, :f3], slice_span].data == view(eeg, [3, 4, 2], span_range).data
+
+# NOTE: Keep in mind that `duration(samples.signal)` is not generally equivalent
```

Comment:
Seems like this should be a `!!! warn` in the docstring for one of the methods of `duration`?

Reply:
Yup added it to the docstring for `Samples` but could be in multiple places because I'm guessing this'll be a common source of confusion